src/linkcheck.sh

```diff
@@ -1,4 +1,6 @@
 #!/bin/sh
+# shellcheck disable=SC2317
```

Comment:
Please keep it local to the affected functions instead of making it global.

Reply:
Done. It was global, as it applies to multiple lines.. but this should be fine now.